python/ray/data/_internal/logical/rules/operator_fusion.py

```diff
@@ -330,18 +356,29 @@ def fused_all_to_all_transform_fn(
         return op
 
 
-def _are_remote_args_compatible(up_args, down_args):
+def _are_remote_args_compatible(prev_args, next_args):
```

Comment:
would it be possible to add unit tests for this function?

Reply:
Good idea. Added in `test_read_map_batches_operator_fusion_compatible_remote_args` and `test_read_map_batches_operator_fusion_incompatible_remote_args`